tests/blackbox/README.md

```diff
@@ -1,11 +1,17 @@
-The intention for the test in this directory are for verifying that a
+The intention for the tests in this directory are for verifying that a
 released, packaged, and installed instance of Stratis is functional.
-Thus the test assumes that the `stratisd` daemon and the `stratis` command
-line are installed, although the test will start the daemon if needed.
+
+The `stratisd` test may be run without the installation of the `stratis` command line.
+
+```bash
+# python3 stratisd_cert.py -v --disk /dev/vdb --disk /dev/vdc --disk /dev/vdd
+```
+
+The stratis-cli test assumes that the `stratisd` daemon is installed, although the test will start the daemon if needed.
 
 Run the test as root and supply 3 or more block devices on the
```

Comment:
These lines are actually true for both, they must both be root and must be supplied with three or more block devices. Try   putting this line at the top or the bottom, so that it does cover both.